Test/COD145904.TXT

```diff
@@ -0,0 +1,186 @@
+OBJECT Codeunit 145904 Test_DotNet_Regex_Toolkit
```

Comment:
This helper codeunit should not exist.

Most of the functions inside can be replaces with functions from the Assert codeunit, the others should be distributed to other test codeunits in this PR?